guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Hammer-cli.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Hammer_CLI_{context}"]
+= Using IdM credentials to log in to the {Project} Hammer CLI
+
+This section describes how to log in to your {Project} Hammer command-line interface (CLI) with your Red{nbsp}Hat Identity{nbsp}Management (IdM) login + password.
```

Comment:
Please use an attribute for "Red{nbsp}Hat Identity{nbsp}Management". See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L61

Also, please replace the "+" with "and".

Reply:
Done